source/_integrations/lutron_caseta.markdown

```diff
@@ -174,6 +168,12 @@ Because Lutron Caseta devices automatically report state to Home Assistant (rath
 
 For more information on working with binary sensors in Home Assistant, see the [Binary Sensors Component](/components/binary_sensor/)
 
+## Button
+
+After setup, buttons will appear in Home Assistant using an `entity_id` based on the name used in the Lutron mobile app. For example, a button 'Button 1' on a keypad called 'Front Entrance Keypad' will appear in Home Assistant as `button.front_entrance_keypad_button_1`.
```

Comment:
Why does this matter?

Reply:
Why do the button entities matter?

For the RA3/HWQSX systems, scenes are not exposed by the system.  However we can apply a scene to a keypad button (physical or virtual).  This will allow Home Assistant to execute these scenes.

Reply:
Is there anything further required to merge this PR?

Reply:
>  Why do the button entities matter?

No, wny it needs to be explained here in this way. Like, the entity IDs shouldn't matter that much (and are above all not guaranteed or fixed), additionally, the naming in Home Assistant should already provide this context, right?

Reply:
> Is there anything further required to merge this PR?

Patience 👍

Reply:
Understood.  I was matching the existing documentation style, but I'll put some thought into a more meaningful entry.

Reply:
Aah, right, I see, that makes sense. I guess a larger part of the documentation is still from an older era 🤔